libraries/AP_HAL_ChibiOS/hwdef/AIRBRAINH743/README.md

```diff
@@ -0,0 +1,93 @@
+
+# AIRBRAINH743 Flight Controller
+
+The AIRBRAINH743 is a flight controller designed and produced by Gear Up.
+
+## Features
+
+ - MCU - STM32H743xx 32-bit processor running at 480 MHz
+ - IMU - Invensense ICM-42688-P
+ - Barometer - DPS368
+ - Magnetometer - LIS2MDL
+ - 128 Mbytes flash-based logging
+ - Battery input voltage: 3S-10S
+ - BEC 5V 2A
+ - BEC 10V 2.5A
+ - 7x UARTs
+ - 9x PWM Outputs (8 Motor Output, 1 LED)
+
+ ## Pinout
+ 
+ ![AIRBRAINH743 Board Top](pinoutTop.png "AIRBRAINH743 Top")
+ ![AIRBRAINH743 Board Bottom](pinoutBottom.png "AIRBRAINH743 Bottom")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB (MAVLink2)
+ - SERIAL1 -> USART1 (Spare)
+ - SERIAL2 -> USART2 (RCIN, DMA-enabled)
+ - SERIAL3 -> USART3 (MSP DisplayPort, DMA-enabled)
+ - SERIAL4 -> UART4 (GPS, DMA-enabled)
+ - SERIAL5 -> UART5 (GPS, DMA-enabled)
+ - SERIAL7 -> UART7 (ESCTelemetry)
+ - SERIAL8 -> UART8 (GPS, DMA-enabled)
+
+## RC Input
+
+The default RC input is configured on USART2. RC could  be applied instead to a different UART port such as EMPTY,  and set
+the protocol to receive RC data ``SERIALn_PROTOCOL`` = 23 and change :ref:`SERIAL2 _PROTOCOL <SERIAL2 _PROTOCOL>`
+to something other than '23'. For rc protocols other than unidirectional, the USART2_TX pin will need to be used:
+
```

Comment:
needs explanation about using UART5 as RC (SBUS) from the HD VTX connector instead of UART2